drafts/25-xxx-specifications.txt

```diff
@@ -206,64 +206,348 @@ to11. The following tokens also appear in the illustrative syntax below.
 3 #-Directives
```

Comment:
Please add `EOL` to the list of tokens appearing in the illustrative syntax. I suggest something like:

```
| EOL                 | The new-line ending a directive line,     |
|                     |  after continuation processing (see li15) |